clustermesh-apiserver/clustermesh/k8s/resource_ctors.go

```diff
@@ -55,5 +71,44 @@ func CiliumEndpointSliceResource(params k8s.CiliumResourceParams, mp workqueue.M
 	)
 	return resource.New[*cilium_api_v2alpha1.CiliumEndpointSlice](params.Lifecycle, lw, mp,
 		resource.WithMetric("CiliumEndpointSlice"), resource.WithCRDSync(params.CRDSyncPromise),
+		resource.WithIndexers(
+			cache.Indexers{
+				cache.NamespaceIndex: cache.MetaNamespaceIndexFunc, // index by namespace for global namespace lookups
+			},
+		),
```

Comment:
This is not correct, because CES resources are cluster scoped, and have a dedicated `Namespace` field.

Reply:
I missed this since `CiliumEndpointSlice` had ObjectMeta embedded. Fixed with a custom indexer function.